backend/LexCore/Exceptions/InvalidEmailException.cs

```diff
@@ -0,0 +1,3 @@
+﻿namespace LexCore.Exceptions;
+
+public class InvalidEmailException(string address) : Exception(address) { }
```

Comment:
I think you can add an additional property. e.g. `public string Email { get; }` that you can pull out of the GQL result. (see e.g. `public DbErrorCode Code { get; }` in `LexData.DbError`)
What do you think about doing that? It would be a bit cleaner than riding on the message property.

Reply:
That's what I'd do too

Reply:
Tried that and it didn't seem to work, but I must have been doing something wrong, because the DbError approach looks very straightforward. Maybe I was getting too elaborate with the Data property of Exception, which is *supposed* to map cleanly onto the GQL `extensions` property but wasn't. I'll go back to a simple property and it'll probably work this time, even though I could have sworn I did it that way originally.

Reply:
Figured out what I did wrong last time. I wrote the property as `string Address { get; }` when I needed to write it as `public string Address { get; }`. I forgot that C# defaults to private if public isn't specified. Once I set it to public, the property started showing up in the autogenerated GraphQL schema and I could reference it.

Reply:
Done in commit cb0ec6e5.